playground/CMakeLists.txt

```diff
@@ -1,22 +1,8 @@
-PROJECT(playground)
+add_executable( stlab.playground main.cpp )
 
-set(SOURCE
-  main.cpp
-)
+target_link_libraries( stlab.playground
+                       PUBLIC ${Boost_LIBRARIES} 
+                       PUBLIC stlab )
 
-set(HEADERS
-)
-
-add_executable(playground ${SOURCE} ${HEADERS})
-
-include_directories(${Boost_INCLUDE_DIRS})
-
-target_link_libraries(
-    playground
-    ${Boost_LIBRARIES} 
-    ${CMAKE_THREAD_LIBS_INIT} 
-    ${CONAN_LIBS})
-
-
-set_property(TARGET playground PROPERTY CXX_STANDARD 14)
-set_property(TARGET playground PROPERTY CXX_STANDARD_REQUIRED ON)
+set_property( TARGET stlab.playground PROPERTY CXX_STANDARD 14 )
```

Comment:
Thanks for the explanation. As I said before, my CMake-fu is pretty weak, so I appreciate you bringing me up to speed on these commands and how they operate.